### DISCUSSION TITLE
Can I customize encoding parameter while exporting video?

### DISCUSSION THREAD

**Discussion by ddozza** [senshu/Sozi#627] [Q&A]
Hello,

I'm using new exporting feature for creating video from Sozi. I wonder to know if it's possible to change encoding parameters while default codec doesn't fit my requirements.

Davide

**Comment by aumouvantsillage** [comment]
It's not possible at the moment.
As a workaround, you can export a presentation to a sequence of images and encode it to a video outside of Sozi.
